src/common/MetaHead.ts

```diff
@@ -0,0 +1,13 @@
+import { IsomerLayout } from "~/engine"
+
+export interface MetaHeadProps {
+  type: "metahead"
+  title: string
+  description?: string
+  noIndex?: boolean
+  favicon?: string
+  layout?: IsomerLayout
```

Comment:
`layout` doesn't seem to be used in `MetaHead.tsx`, and tbh yea I don't understand why it would need to be in the props

Reply:
Yeah oops actually this one can remove. I originally had it for a different component called `MetaScripts` (which is the same thing [but for scripts](https://nextjs.org/docs/pages/api-reference/components/script)), which will need to adjust the script imports based on the type of layout that the page is using.

For reference it will look something like this: https://github.com/isomerpages/isomerpages-template/blob/next-gen/_includes/main_scripts.html

Reply:
Removed in f7dc591254749236e4d8c8ce713560be5a68d263.